addons/binding/org.openhab.binding.draytonwiser/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,429 @@
+        <category>Temperature</category>
+        <state readOnly="false" pattern="%.1f °C"></state>
+    </channel-type>
+    
+    <channel-type id="batteryVoltage-channel" advanced="true">
+        <item-type>Number</item-type>
+        <label>Battery Voltage</label>
+        <description>Current Battery Voltage for the device</description>
+        <state readOnly="true" pattern="%.1f V"/>
+    </channel-type>
+    
+    <channel-type id="demandPercent-channel">
+        <item-type>Number</item-type>
+        <label>Current Heat Demand</label>
+        <description>Current heat demand in %</description>
+        <state readOnly="true" pattern="%d %%"></state>
+    </channel-type>
+    
+    <channel-type id="heatRequest-channel">
+        <item-type>Switch</item-type>
+        <label>Currently Requesting Heat</label>
+        <description>Is this room requesting heat</description>
+        <state readOnly="true" ></state>
+    </channel-type>
+    
+    <channel-type id="heatingOverride-channel">
+        <item-type>Switch</item-type>
+        <label>Heating Override</label>
+        <description>The heating override button has been pressed</description>
+        <state readOnly="true" ></state>
+    </channel-type>
+    
+    <channel-type id="hotWaterOverride-channel">
+        <item-type>Switch</item-type>
+        <label>Hot Water Override</label>
+        <description>The hot water override button has been pressed</description>
+        <state readOnly="true" ></state>
+    </channel-type>
+    
+    <channel-type id="signalRSSI-channel" advanced="true">
+        <item-type>Number</item-type>
+        <label>Signal RSSI</label>
+        <description>The reported network signal RSSI</description>
+        <state readOnly="true" ></state>
+    </channel-type>
+    
+    <channel-type id="signalLQI-channel" advanced="true">
+        <item-type>Number</item-type>
+        <label>Signal LQI</label>
+        <description>The reported network signal LQI</description>
+        <state readOnly="true" ></state>
+    </channel-type>
+    
+    <channel-type id="signalStrength-channel" advanced="true">
+        <item-type>String</item-type>
+        <label>Signal Strength</label>
+        <description>The reported network signal strength</description>
+        <state readOnly="true" ></state>
+    </channel-type>
+    
+    <channel-type id="unifiedSignalStrength-channel">
+        <item-type>Number</item-type>
+        <label>Unified Signal Strength</label>
+        <description>The reported network signal strength</description>
+        <state readOnly="true" ></state>
+    </channel-type>
+    
+    <channel-type id="batteryLevel-channel" advanced="true">
+        <item-type>String</item-type>
+        <label>Battery Level</label>
+        <description>Current Battery Level for the device</description>
+        <state readOnly="true"/>
+    </channel-type>
+    
+    <channel-type id="unifiedBatteryLevel-channel">
```

Comment:
What is the purpose of this additional channel if you already have a battery-level?

Reply:
This is a good question. The batteryLevel channel exposes the drayton battery levels of "Good, Poor" etc, whereas the unifiedBatteryLevel exposes a percentage so you can use them in sitemaps.
Same with the signalStrength and unifiedSignalStrength.
I exposed them as I initially though someone might want to actually see the "drayton" version of these values, but I'd appreciate your opinion on this.